controllers/storagecluster/cephrbdmirrors.go

```diff
@@ -3,20 +3,29 @@ package storagecluster
 import (
 	"context"
 	"fmt"
+	"os"
+	"time"
 
 	ocsv1 "github.com/red-hat-storage/ocs-operator/api/v1"
 	"github.com/red-hat-storage/ocs-operator/controllers/defaults"
 	cephv1 "github.com/rook/rook/pkg/apis/ceph.rook.io/v1"
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/klog/v2"
+	"k8s.io/utils/pointer"
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 )
 
 type ocsCephRbdMirrors struct{}
 
+var (
+	rbdMirrorDebugLoggingEnabled = false
```

Comment:
This var would be reset each time the OCS operator is restarted. So we will run the job again each time the operator is restarted?

Reply:
Even if the var is reset, it will try to create the job again, So there it will encounter an alreadyExists error, Then it will check if the job was successful. If it was, then it will set the var to true again. So I think we are safe.